docs/OpenCLIExamples.rst

```diff
@@ -0,0 +1,346 @@
+==========================
+CLI Open Endpoint Examples
+==========================
+
+The HCA CLI provides users of the Human Cell Atlas (HCA) to access and download data sets from the HCA. This page
+covers how to access the HCA using the HCA command line utility.
+
+*NOTE:* The HCA CLI utility is compatible with Python 3.5+.
+
+
+``hca create-version``
+----------------------
+
+Returns a timestamp in ``DSS_VERSION`` format (e.g., ``1985-04-12T232050.520000Z``), useful for
+versioning bundles or files.
+
+.. note::
+  A version is a timestamp in RFC3339 format that keeps track of the most recent iteration of a
+  bundle or file. A bundle consists of many different data files, and both bundles and files have
+  version numbers.
+
+Example call to ``hca create-version``:
+
+.. literalinclude:: ../test/tutorial/scripts/cli/create_version_cli.sh
+
+``hca download``
```

Comment:
The CLI and API descriptions seem to be duplicated.  Is there a way to consolidate them (or at least the shared portions of the description) into one location (imported into the docs)?

Reply:
Yes, that's possible using `:: include path/to/file.rst` in Sphinx. See f929343 for example.

Reply:
Perhaps we can leave the work of splitting up the sections into transcluded documents for a future PR?